doc/application-maintainer.rst

```diff
@@ -0,0 +1,141 @@
+==========================================
+Maintaining Neptune-Versioned Applications
+==========================================
+
+This guide is intended to help having a common ground when versioning
+applications, in particular applications that are deployed to provide services,
+but it could probably be extended to other type of applications. The goal is to
+let users and people affected by changes in applications quickly know what to
+expect on new releases.
+
+Since applications have very different impact and some are more critical than
+others, it is very hard to have a strict guideline like with `libraries
+<library-maintainer.rst>`_, so this is just a guideline and each project must
+adapt it to what works best for it.
+
+.. contents::
+
+
+Version Numbers
+===============
+
+Numbering versions of releases should follow `SemVer 2.0 <http://semver.org/>`_.
+This only means, a version number should have the form::
+
+  X.Y.Z[-P][+M]
+
+Where:
+
+X
+  is the **Major** version
+Y
+  is the **Minor** version
+Z
+  is the **Patch** version
+P
+  is a pre-release identifier (optional)
+M
+  is build metadata (optional)
+
+For more details on which kind of characters can be used on each, please
+consult the SemVer guide. The meaning of these numbers, though, does not
```

Comment:
OK, what about something like "because SemVer is thought for APIs, so all wording is specific to them and applications deal with other kind of interfaces (network protocols, end users, etc.)"?